### DISCUSSION TITLE
Uncaught ReferenceError: isRecordSelected is not defined

### DISCUSSION THREAD

**Discussion by markgoedegebure** [filamentphp/filament#9806] [Help] [answered]
I have just upgraded from Filament v2 to v3. The tables are showing correctly, but all interaction is not working. I can't sort the columns, open the filters of switch to the next page. 
In my console i see the following errors:
![image](https://github.com/filamentphp/filament/assets/44771394/8eff7e8b-1634-4319-80f3-a3aafeb2319b)
![image](https://github.com/filamentphp/filament/assets/44771394/6f8b9de1-aeb0-4b5b-88e5-e8aacc1c1a66)

My app.js:
![image](https://github.com/filamentphp/filament/assets/44771394/ed1f3203-efee-4980-b381-3a9bb82954fd)

My app.blade.php
![image](https://github.com/filamentphp/filament/assets/44771394/d63966d9-9044-4ebb-a44d-d64c2489d379)

My table class:
![image](https://github.com/filamentphp/filament/assets/44771394/1fe45d3b-bb3b-4ff6-ad21-5112ded8f171)

Does anyone know what might cause these errors? I can't find anything related to isRecordSelected

**Comment by leandrocfe** [comment]
Are you using panel builder or only the table builder?

**Reply by markgoedegebure** [reply]
I have the full Filament package installed, but only using the table builder at the moment.

**Reply by leandrocfe** [reply]
Hum.. the asset management system is different in V3. Have a look
[assets](https://filamentphp.com/docs/3.x/support/assets)
[custom-theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)

Maybe this is the issue..

Are you using table builder with custom livewire components? Have a look [adding-the-table](https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component#adding-the-table)

**Reply by markgoedegebure** [reply]
i don't see how these assets are related to my case. The upgrade guide or installation guide doesn't mention anything about this. 
I just have a Livewire component that implements HasTable. I don't really know or understand what to add or implement in that FilamentAsset::register() function of why the upgrade and installation guides doesn't mention anything about this.

I added my table class in this discussion above

**Reply by leandrocfe** [reply]
*full Filament package installed*, you mean [panel builder](https://filamentphp.com/docs/3.x/panels/installation) or did you install table builder, form builder, etc?

**Reply by markgoedegebure** [reply]
I have the panel builder installed: `composer require filament/filament:"^3.0-stable" -W`
This has the table builder, form builder, etc pre-installed

**Reply by leandrocfe** [reply]
Did you publish filament views?

**Reply by markgoedegebure** [reply]
Yes, i have published the views

**Reply by leandrocfe** [reply]
try to remove them to test it

**Reply by markgoedegebure** [reply]
unfortunately removing doesn't make any difference. Republishing neither

**Reply by leandrocfe** [reply]
Are you able to share this project on the Github, without the data, resources, only one page just to try this behavior?

**Reply by obrunsmann** [reply]
I have the same problem. I have longer experience with Filament (v2+v3) and can confirm that it only occurs for me in conjunction with wire:navigate.

**Reply by markgoedegebure** [reply]
i just checked, but i'am not using wire:navigate anywhere in my application. I could be used in a package, but then more peaple should experience this issue, right? 
This problem occurs with all my Filamwnt tables, not just one.
I didn't have this problem with Filament v2

**Reply by leandrocfe** [reply]
Try to share one example on github repo. Maybe I can help you

**Reply by markgoedegebure** [reply, marked as answer]
The problem is solved. Apparently it is required in filament v3 to have a root element around {{ $this->table }} in the view. This wasn't required in the v2. After adding a div, the tables work as expected. Strange this is nowhere noted in the docs/upgrade guide.

**Reply by Binyamkpaul** [reply]
i have the same problem but its A normal Filament Resource but when i am using with proxy like https://away.domain.et/proxy/6cf8415f/https/somain.domain.et/ any solution
